hal/transport/RS485/MyTransportRS485.cpp

```diff
@@ -267,14 +260,7 @@ bool transportSend(const uint8_t to, const void* data, const uint8_t len, const
 		}
 	}
 
-#if defined(MY_RS485_DE_PIN)
```

Comment:
Can we just stop checking MY_RS485_DE_INVERSE and MY_RS485_DE_PIN? Or do those checks be part of (de)assertDE?

Reply:
This is crucial for correct transmission. If MY_RS485_DE_PIN is defined, it must be set to HIGH when transmitting and to receive anything it must be set to LOW. If we also define MY_RS485_DE_INVERSE , it means that this logic is inverted. If the pin is not defined, it means that switching between transmitting and receiving must be handled differently, e.g. by a transceiver. Most low-cost transceivers don't do that. 
BTW this part of pull request is just a code cleanup and has no effect on the object code or program operation. Just previously defined macros have been used.

Reply:
This PR removes the two if clauses and replaces them with the macro. The macro does not contain the if clauses. Won't that result in non-working devices?

Reply:
That's because this macro is defined differently depending on whether or not  MY_RS485_DE_PIN and MY_RS485_DE_INVERSE  are defined.
https://github.com/klaudiusz223/MySensors/blob/e70973144ee59f3c86616ff15f446fafa905340f/hal/transport/RS485/MyTransportRS485.cpp#L59-L70.

Reply:
Great, thanks for explaining!